src/components/AccountModule/connection-statuses.js

```diff
@@ -0,0 +1,16 @@
+export const CONNECTION_STATUS_ERROR = Symbol('CONNECTION_STATUS_ERROR')
+export const CONNECTION_STATUS_HEALTHY = Symbol('CONNECTION_STATUS_HEALTHY')
+export const CONNECTION_STATUS_WARNING = Symbol('CONNECTION_STATUS_WARNING')
```

Comment:
Would it make sense to name these variables similarly, e.g. `STATUS_CONNECTION_ERROR`?